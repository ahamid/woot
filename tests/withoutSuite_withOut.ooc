main: func() {
    "ooc-file without suite config but with out document" println()
}

