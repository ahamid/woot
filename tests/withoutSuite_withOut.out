ooc-file without suite config but with out document
